tests/generic_config_updater/test_vlan_interface.py

```diff
@@ -95,10 +174,10 @@ def vlan_interface_tc1_xfail(duthost):
     ("remove", "Vlan1000", "192.168.0.1/21", "fc02:1000::2/64") REMOVE Unexist IPv6 address
     """
     xfail_input = [
-        ("add", "Vlan1000", "587.168.0.1/21", "fc02:1000::1/64"),
-        ("add", "Vlan1000", "192.168.0.1/21", "fc02:1000::xyz/64"),
-        ("remove", "Vlan1000", "192.168.0.2/21", "fc02:1000::1/64"),
-        ("remove", "Vlan1000", "192.168.0.1/21", "fc02:1000::2/64")
+        ("add", vlan_info["v4"]["name"], reg_replace(vlan_info["v4"]["prefix"], r"^\d*", "587"), vlan_info["v6"]["prefix"]),
```

Comment:
Why not use 587.168.0.1/21 directly? This change will cause the test inconsistence with doc (LN 171 - 174)

Reply:
Vlan prefix len in different topo possible different, so I think dynamic replacement by regex is more flexible than directly use